pages/index.tsx

```diff
@@ -16,7 +16,7 @@ const Home: PageWithLayout = () => {
   }, []);
 
   return (
-    <div className="flex flex-col p-6 items-center justify-center 2xl:justify-around min-h-screen min-w-full bg-black text-white">
+    <div className="flex flex-col p-6 items-center justify-center 2xl:justify-around min-h-screen min-w-full text-white">
```

Comment:
This looks good, I set a blue background on layout component for debugging under `./lib/components/Layouts/Layout.tsx` which you can also remove.